e2e/local-cluster-config.yaml

```diff
@@ -0,0 +1,8 @@
+# four nodes (three workers) cluster config
+kind: Cluster
```

Comment:
I just noticed that we already have this setup: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/scripts/setup_kind_local_registry.sh do you think it make sense to combine those to prevent the number of different kind scripts?